docs/debezium-aws-rds-config-scripts

```diff
@@ -0,0 +1,361 @@
+Give permission to Port Number ***TCP:5432*** to all sources (0.0.0.0/0) via ***DB Security Group***.
+
+![m'lady](./images/AWS-Security-Group-permission.png) 
+
+4. Connect to the database via psql 
+
+----
+    [ec2-user@hostname ~] $ psql -h testdbserver.cuwbhrbzgqk9.us-east-1.rds.amazonaws.com -d testdb -U testdbuser -p 5432
+----
+
+5. Type \d to see a screen like that and be sure that  your user testdbuser is a member of rds_superuser and rds_superuser has ***rds_replication*** right
+
+    `testdb=> \du`
+    
+	![m'lady](./images/Replication-RDS.png) 
+	
+
+6. Display if any replication slots are available
+
+    `testdb=> select * from pg_replication_slots;`
+
+7. Create a replication plugin called ***wal2json*** and name it ***wal2json_rds***
+
+    `testdb=> SELECT * FROM pg_create_logical_replication_slot('wal2json_rds', 'wal2json');`
+
+8. If you wanna delete the slot type this
+    
+    `testdb=> SELECT pg_drop_replication_slot('wal2json_rds');`
+
+9. Create a table named ***customers***
+
+    `testdb=> CREATE TABLE customers (id SERIAL,  first_name VARCHAR(255) NOT NULL,  last_name VARCHAR(255) NOT NULL,  email VARCHAR(255) NOT NULL,  PRIMARY KEY(id));`
+
+10. Change table so that it can ***support replication***
+
+    `testdb=> ALTER TABLE customers REPLICA IDENTITY FULL;`
+
+11. ***Insert some records*** into the source table
+
+----
+    testdb=> insert into customers values(default, 'fatih1', 'kose1', 'fatih1.kose@example.com');
+    testdb=> insert into customers values(default, 'fatih2', 'kose2', 'fatih1.kose@example.com');
+----
+
+### 1.3 Prepare Debezium Test Environment And Complete Configuration On Linux
+
+1. Create a directory under user home directory 
+
+    `[ec2-user@hostname ~] $ mkdir ~/debezium`
+
+2. Download the sample config scripts from ***https://github.com/fatihmehmetkose/debezium-aws-rds-config-scripts*** and copy them into that directory
+
+    `[ec2-user@hostname ~] $ cp *.zip debezium/`
+
+3. Enter into that directory
+
+    `[ec2-user@hostname ~] $ cd debezium`
+
+4. ***export DEBEZIUM_VERSION*** environment variable
+
+    `[ec2-user@hostname ~] $ export DEBEZIUM_VERSION=0.7.1`
+
+5. ***Build docker-compose files*** named docker-compose.yaml (default name)
+
+    `[ec2-user@hostname ~] $ docker-compose build`
+
+6. ***Start the docker*** containers. Applications running on containers are ***Zookeper, Apache Kafka, Debezium Kafka Connector, PostgreSQL db***. 
+
+    `[ec2-user@hostname ~] $ docker-compose up`
+
+7. Create a table named ***customers*** on target or sink PostgreSQL db
+
+    `[ec2-user@hostname ~] $ docker-compose -f docker-compose.yaml exec postgres bash -c 'psql -U $POSTGRES_USER $POSTGRES_DB -c "CREATE TABLE customers (id SERIAL,  first_name VARCHAR(255) NOT NULL,  last_name VARCHAR(255) NOT NULL,  email VARCHAR(255) NOT NULL,  PRIMARY KEY(id));"'`
+
+8. ***Start target PostgreSQL DB connector*** running in a container
```

Comment:
We should clarify which connector this is. I reckon it's the Confluent JDBC sink connector?

Reply:
Ok